.gitpod.yml

```
@@ -11,8 +11,8 @@ ports:
   onOpen: ignore
 tasks:
 - name: Start App
-  # run chown because https://github.com/gitpod-io/gitpod/issues/4851
-  before: sudo chown -R gitpod:999 /workspace/openlibrary/
+  # because: https://github.com/gitpod-io/gitpod/issues/9311
+  before: chmod 0755 $GITPOD_REPO_ROOT
```

```suggestion
  before: |
    # run chown because https://github.com/gitpod-io/gitpod/issues/4851
    sudo chown -R gitpod:999 /workspace/openlibrary/
    # because: https://github.com/gitpod-io/gitpod/issues/9311
    chmod 0755 $GITPOD_REPO_ROOT
```

Eg something like this?